### PR TITLE
Move /metrics endpoint to separate port

### PR DESCRIPTION
**Pull request by SpontaneousOverthrow**
## The Problem
`/metrics` endpoint and `/admin-enpoint` both are listening on the same port. 
## The Solution
Move `/metrics` endpoint to separate port, this will increase security

## Checklist

- [ ] Test manually on `test-ui` using remote chains.
- [ ] Run load tests.
- [ ] Update documentation if needed.
- [ ] Update CHANGELOG.md.

### CODE REVIEW

**Comment by SpontaneousOverthrow** [comment]
Ready for testnet

**Review by rhlsthrm — APPROVED** [review]

File: packages/router/src/bindings/fastify/index.ts [hunk 1]
```diff
@@ -195,12 +184,25 @@ export const bindFastify = () =>
...
+    server.listen(config.metrics_port, config.metrics_host, (err, address) => {
       if (err) {
         console.error(err);
         process.exit(1);
       }
       logger.info(`Server listening at ${address}`);
       res();
     });
+
+    server.get("/metrics", async (request, response) => {
```
[the remark this answers is not in the document]
Comment (SpontaneousOverthrow) [reply packages/router/src/bindings/fastify/index.ts:196]: > Does this work? I would think we would need two separate server instances.

Need to test, I can build image and test manually tomorrow.

[the remark this answers is not in the document]
Comment (SpontaneousOverthrow) [reply packages/router/src/bindings/fastify/index.ts:196]: > Does this work? I would think we would need two separate server instances.

does not work, need to create separate bindFastify

[the remark this answers is not in the document]
Comment (rhlsthrm) [reply packages/router/src/bindings/fastify/index.ts:196]: OK, I thought so!

**Pull request merged**